Admin/Admin.php

```diff
@@ -416,6 +416,13 @@
     protected $breadcrumbs = array();
 
     /**
+     * The breadcrumbsBuilder component.
+     *
+     * @var BreadcrumbsBuilderInterface
+     */
+    private $breadcrumbsBuilder;
```

Comment:
I'll fix this, but please note that other properties in this file do not respect this standard. Also, I put it here to group it with the `breadcrumbs` property.